executors/Cargo.toml

```diff
@@ -24,27 +24,30 @@ workstealing-exec = ["crossbeam-channel", "crossbeam-deque", "rand", "crossbeam-
 defaults = ["num_cpus"]
 
 # In the workstealing executor, check the global queues every 1ms
-ws-timed-fairness = ["time"]
+ws-timed-fairness = [] #["time"]
 # Otherwise check the global queues every 100 jobs
 
 # In the workstealing executor, never park worker threads
 ws-no-park = []
 
+thread-pinning = ["core_affinity"]
+
 
 [dependencies]
 log 				= "0.4"
 synchronoise 		= "1.0"
 arr_macro 			= "0.1.2"
-crossbeam-channel 	= {version = "0.3", optional = true}
+crossbeam-channel 	= {version = "0.4", optional = true}
 threadpool 			= {version = "1.7", optional = true}
-crossbeam-utils 	= {version = "0.6", optional = true}
+crossbeam-utils 	= {version = "0.7", optional = true}
 crossbeam-deque 	= {version = "0.7", optional = true}
-time 				= {version = "0.1", optional = true}
+#time 				= {version = "0.2", optional = true, default-featurs = false, features = ["std", ""]}
```

Comment:
`default-featurs` should be `default-features` :)

Reply:
True^^ It's my damn dying MacBook keyboard that swallows e's -.-
Anyway, the line is commented out and I should probably just delete it at some point, since I'm not depending on the time crate anymore.